source/documentation/deploying-an-app/using-circleci-for-continuous-deployment.html.md.erb

```diff
@@ -48,22 +48,21 @@ metadata:
   name: circleci
   namespace: <YOUR-NAMESPACE>
 rules:
-  - apiGroups: ["", "extensions", "apps", "batch"]
+  - apiGroups:
+      - ""
     resources:
       - "pods/portforward"
       - "deployment"
       - "secrets"
       - "services"
       - "pods"
-      - "jobs"
```

Comment:
Did you mean to remove "jobs" from this list?

Reply:
Thanks @digitalronin, added jobs now